src/templates_front/templator_front.org

```diff
@@ -1315,7 +1319,7 @@ def _inquire(file_name: str) -> bool:
 
    For each of the aforementioned objects, TREXIO provides *has*,
    *read* and *write* functionality.  TREXIO supports I/O with single
-   or double precision for integer and floating point numbers.
+   or real(c_double) for integer and floating point numbers.
```

Comment:
I think this is an instance of undesired search-and-replace?  I think in comments this should continue to read "double precision".  There are a few more throughout.

Reply:
True!